frontend/src/features/books/BookDetails.jsx

```diff
@@ -1,27 +1,22 @@
 import React, { useEffect, useState } from 'react'
-import { useParams, Link } from 'react-router-dom'
+import { useParams, Link, useNavigate } from 'react-router-dom'
 import { API_URL } from '../../constants'
 
 function BookDetails() {
-  const [title, setTitle] = useState(null)
-  const [author, setAuthor] = useState(null)
-  const [description, setDescription] = useState(null)
+  const [book, setBook] = useState(null)
```

Comment:
@nshelokar-vf, is there any specific reason why we are not using `;` at the end of line?

Reply:
I forgot to add it and was not aware about that ' ; ' make big impact in later stage but now I am trying to focus on every detail of coding and syntax.